release/ray_release/byod/byod_rllib_test.sh

```diff
@@ -16,7 +16,7 @@ mkdir ~/.mujoco
 mv mujoco210-linux-x86_64.tar.gz ~/.mujoco/.
 cd ~/.mujoco
 tar -xf ~/.mujoco/mujoco210-linux-x86_64.tar.gz
-echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/ray/.mujoco/mujoco210/bin' >> /home/ray/.bashrc
+echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:/home/ray/.mujoco/mujoco210/bin" >> /home/ray/.bashrc
```

Comment:
hmm.... you actually meant to not escape huh.. not sure if there is some way to tell the linter to ignore this.